README.md

```diff
@@ -21,6 +21,35 @@ To use Jenres, you need to follow the following procedure:
 
 ⚠️ Verify the data that will be shared with the Jenres APIs. Make sure the URL is of the type `https://jenres-api.aws.res-it.com`.
 
-At this point, you can start using Jenres by creating the first issue and naming @jenres in the issue.
+
+### Application examples
+
+#### listen function
+
+It is possible to solve your git issue with **jenres listen**. In order to do that, create a new issue with a title and naming `@jenres` in the issue content. A pull request will be opened by jenres at end of the elaboration.
```

Comment:
Good, a few notes:

- jenres -> Jenres
- jenres listen is an internal subcommand, but the user doesn't know it is a name for a functionality, let's call it "Jenres Listen" with upper case so they understand